lib/chef/provisioning/convergence_strategy/precreate_chef_objects.rb

```diff
@@ -31,6 +31,30 @@ def setup_convergence(action_handler, machine)
         chef_server_url = chef_server[:chef_server_url]
         chef_server_url = machine.make_url_available_to_remote(chef_server_url)
 
+        convergence_options[:remote_forwards].each_pair do |local_url, path_or_config|
```

Comment:
Because this only pertains to SSH (winrm does not support port forwarding) we should make this part of the `ssh_options` machine_option and move it into the SSH transport class.

Note:  I think the `ssh_options` are passed to the SSH transport class as the `options` parameter